caffe2/python/operator_test/elementwise_op_broadcast_test.py

```diff
@@ -237,8 +238,66 @@ def test_sum_reduce(self, gc, dc):
         out = workspace.FetchBlob("out")
         res = np.array(np.sum(X))
         np.testing.assert_array_almost_equal(out, res, decimal=0)
-        self.assertDeviceChecks(dc, op, [X, Y], [0])
 
+        # fp64 is not supported with the CUDA op
+        dc_cpu_only = [d for d in dc if d.device_type != caffe2_pb2.CUDA]
+        self.assertDeviceChecks(dc_cpu_only, op, [X, Y], [0])
+
+    @given(**hu.gcs_gpu_only)
```

Comment:
We need to add
@unittest.skipIf(not workspace.has_gpu_support, "No gpu support.")

otherwise this fails on non-GPU machine

Reply:
Sorry, I don't often remember to test that, and TravisCI wasn't helpful until recently. Fixed.